app/controllers/AdministratorController.java

```diff
@@ -20,4 +27,9 @@
 	public void setService(final AdministratorService service) {
 		this.service = service;
 	}
+	
+	@Transactional(readOnly = true)
+	public Result getStatistics() {
+		return wrapForPublic(() -> this.service.getStatistic());
```

Comment:
"Administrator" statistics infers that they should only be visible to administrators. `wrapForPublic` does not require a logged in user.

Reply:
U redu.